core/src/main/java/org/sonarsource/sonarlint/core/WsHelperImpl.java

```diff
@@ -43,13 +44,15 @@
 import org.sonarsource.sonarlint.core.util.StringUtils;
 import org.sonarsource.sonarlint.core.util.ws.WsResponse;
 
+// TODO
 import static com.google.common.base.Preconditions.checkNotNull;
 
 public class WsHelperImpl implements WsHelper {
   private static final String MIN_VERSION_FOR_ORGANIZATIONS = "6.3";
 
   @Override
   public ValidationResult validateConnection(ServerConfiguration serverConfig) {
+      if (true) System.out.println("foo");
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Replace this use of System.out or System.err by a logger. ([squid:S106](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS106))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarlint.core%3Asonarlint-core-parent&branch=junk2&issues=AWEyAmflzwlWAzALxAcx&open=AWEyAmflzwlWAzALxAcx)

[ISSUE_KEY]: AWEyAmflzwlWAzALxAcx